go.mod

```diff
@@ -227,7 +225,7 @@ replace (
 	github.com/cosmos/ledger-cosmos-go => github.com/cosmos/ledger-cosmos-go v0.12.4
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
-	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.35.0-tm-v0.34.29
+	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.36.0-tm-v0.34.29
```

Comment:
Just to double check:

@evan-forbes is it safe to backport https://github.com/celestiaorg/celestia-core/pull/1290 to celestia-app v1.x? It was marked as breaking.

Reply:
I spoke with Evan about it earlier and seems to be fine. It's getting rid of the old tracer, the config stays the same. Evan can confirm tho

Reply:
thanks for the info. I'm curious what happens if a user doesn't regenerate their config and upgrades from v1.9.0 -> a binary with this bump. I just tried it and my node was able to continue producing blocks so I think this is safe.

I'm guessing the tracing feature won't work until the user re-generates their config files via `celestia-appd init`. Is that right?

Reply:
yeah, this is correct. it breaks the old tracer as a feature, doesn't break from a config perspective.